### PR TITLE
Improve German README

### PR DESCRIPTION
**Pull request by mybearworld**
Use proper grammar/orthography and translate the shields.

### CODE REVIEW

File: READMEde.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
 
-## API Dokumentierung <!-- ## API documentation -->
+## API-Dokumentation <!-- ## API documentation -->
 
 Dritte dürfen diese öffentliche API mit folgenden Restriktionen benützen:
 
-- **Namensnennung**: Dieses Projekt sollte klar und mit folgendem Link [returnyoutubedislike.com](https://returnyoutubedislike.com/) versehen zugeordnet werden.
-- **Raten Limitierung**: Es bestehen pro Nutzer limitierungen von 100 pro Minute sowie 10'000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass sich Ihre Anwendung zurückziehen sollte.
+- **Namensnennung**: Dieses Projekt sollte klar mit folgendem Link [returnyoutubedislike.com](https://returnyoutubedislike.com/) versehen werden.
+- **Ratenlimitierung**: Es bestehen pro Nutzer Limitierungen von 100 Anforderungen pro Minute sowie 10.000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass sich Ihre Anwendung zurückziehen sollte.
```
Comment (tsomic) [inline READMEde.md:42]:
```suggestion
- **Ratenlimitierung**: Es bestehen pro Nutzer Limitierungen von 100 Anfragen pro Minute sowie 10.000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass sich Ihre Anwendung zurückziehen sollte.
```

File: READMEde.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
 https://returnyoutubedislikeapi.com
 
 Eine Liste aller verfügbaren Endpunkten finden Sie hier:  
 https://returnyoutubedislikeapi.com/swagger/index.html
 
-### Votum erhalten <!-- ### Get votes -->
+### Bewertung erhalten <!-- ### Get votes -->
 
-Beispiel um die Abstimmungen einer gegebenen YouTube ID zu erhalten:  
+Beispiel um die Bewertung einer gegebenen YouTube ID zu erhalten:
```
Comment (tsomic) [inline READMEde.md:52]:
```suggestion
Beispiel, um die Bewertung einer gegebenen YouTube ID zu erhalten:  
```

File: READMEde.md [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
 [FAQ](https://github.com/Anarios/return-youtube-dislike/blob/main/Docs/FAQ.md)
 
 ## Warum dies wichtig ist <!-- ## Why it Matters -->
 
 Mehr erfahren Sie auf unserer Website unter: [returnyoutubedislike.com](https://www.returnyoutubedislike.com/)
 
-## API Dokumentierung <!-- ## API documentation -->
+## API-Dokumentation <!-- ## API documentation -->
 
 Dritte dürfen diese öffentliche API mit folgenden Restriktionen benützen:
```
Comment (tsomic) [inline READMEde.md:39]:
```suggestion
Dritte dürfen diese öffentliche API unter folgenden Restriktionen verwenden:
```

**Review by tsomic — APPROVED** [review]
Great work! I'm also a native German speaker

File: READMEde.md [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
-## API Dokumentierung <!-- ## API documentation -->
+## API-Dokumentation <!-- ## API documentation -->
 
-Dritte dürfen diese öffentliche API mit folgenden Restriktionen benützen:
+Dritte dürfen diese öffentliche API unter folgenden Restriktionen verwenden:
 
-- **Namensnennung**: Dieses Projekt sollte klar und mit folgendem Link [returnyoutubedislike.com](https://returnyoutubedislike.com/) versehen zugeordnet werden.
-- **Raten Limitierung**: Es bestehen pro Nutzer limitierungen von 100 pro Minute sowie 10'000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass sich Ihre Anwendung zurückziehen sollte.
+- **Namensnennung**: Dieses Projekt sollte klar mit folgendem Link [returnyoutubedislike.com](https://returnyoutubedislike.com/) versehen werden.
+- **Ratenlimitierung**: Es bestehen pro Nutzer Limitierungen von 100 Anfragen pro Minute sowie 10.000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass sich Ihre Anwendung zurückziehen sollte.
```
Comment (StarAppeal) [inline READMEde.md:42]:
```suggestion
- **Ratenlimitierung**: Es bestehen pro Nutzer Limitierungen von 100 Anfragen pro Minute sowie 10.000 pro Tag. Dies wird eine Statusmeldung von _429_ ausgeben, die darauf hinweist, dass Ihre Anwendung eine Pause einlegen sollte.
```

Comment (mybearworld) [reply READMEde.md:42]: That changes the meaning from the original.

Comment (StarAppeal) [reply READMEde.md:42]: Yes, it does slightly, but in my opinion it sounds more natural than "Ihre Anwendung muss sich zurückziehen".

**Pull request closed without merging**